koku/providers/azure/provider.py

```diff
@@ -142,7 +142,6 @@ def cost_usage_source_is_reachable(self, credential_name, storage_resource_name)
             key = ProviderErrors.AZURE_BILLING_SOURCE_NOT_FOUND
             raise ValidationError(error_obj(key, str(costreport_err)))
         except (AdalError, AzureException, AzureServiceError, ClientException, TypeError) as exc:
-            LOG.error(f"Error type: {str(exc)}")
```

Comment:
Do we want to capture the log as warning vs. just removing it?

Reply:
Just pushed a change to log the internal and external messages.